packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import Text from '../text';
 
 // Todo: Create a utility type for getting a range of float number. ex: FloatRange<0, 1>
-type TRangeFloatZeroToOne = 0 | 0.1 | 0.2 | 0.3 | 0.4 | 0.5 | 0.6 | 0.7 | 0.8 | 0.9 | 1;
+export type TRangeFloatZeroToOne = 0 | 0.1 | 0.2 | 0.3 | 0.4 | 0.5 | 0.6 | 0.7 | 0.8 | 0.9 | 1;
```

Comment:
@sergei-deriv No need to export, Please use `React.ComponentProps` generic.

Reply:
> @sergei-deriv No need to export, Please use `React.ComponentProps` generic.

I've added this type in 'open-positions' component:
type TRangeFloatZeroToOne = React.ComponentProps<typeof ProgressBar>['value'];